Core/Core/Router/RouteHandler.swift

```diff
@@ -61,7 +61,7 @@ public struct RouteHandler {
                 guard parts.removeFirst() == template else { return nil }
             case .param(let name):
                 guard !parts.isEmpty else { return nil } // too short
-                params[name] = String(parts.removeFirst())
+                params[name] = ID.expandTildeID(String(parts.removeFirst()))
```

Comment:
I am a little bit worried about `Page` slugs that look like a shard id because this will break looking them up. The odds of that seem pretty low though.. What do you think? Should we be worried about that?

Reply:
yeah that's possible. A page slug could be expanded because it had a tilde between 2 numbers.